src/components/metrics.tsx

```diff
@@ -452,6 +479,21 @@ export const QueryTable: React.FC<QueryTableProps> = ({ index, namespace }) => {
           >
             {isDisabledSeriesEmpty ? t('Unselect all') : t('Select all')}
           </Button>
+          <Button
+            variant="link"
+            isInline
+            icon={<ExternalLinkSquareAltIcon />}
+            iconPosition="right"
+            className="query-browser__table-export-link"
+          >
+            <CSVLink
+              data={csvData}
+              filename={`OpenShift_Metrics_QueryResultsTable_${query.replace(/\s/g, '')}.csv`}
+              enclosingCharacter={``}
```

Comment:
```suggestion
              enclosingCharacter=""
```

Reply:
In the latest commit, removed the `react-csv` library components and rewrote the PR. I couldn't get the library to handle some data syncing...so I just decided to remove the library. Here's the snippet that formats the data items with the enclosingCharacters `""`.

```
  const getCsv = (array, delimiter = ',') =>
    array
      .map((row) =>
        row.map((rowItem) => (isNaN(rowItem) ? `"${rowItem}"` : rowItem)).join(delimiter),
      )
      .join('\n');
```
> `"${rowItem}"` -- wraps strings with the enclosingCharacters `""`